apps/nextjs-website/src/components/atoms/LinkButton/LinkButton.tsx

```diff
@@ -4,13 +4,35 @@ import Link from 'next/link';
 import React from 'react';
 
 type LinkButtonProps = {
-  readonly href: string;
+  readonly href?: string;
   readonly label: string;
   readonly size?: number;
   readonly width?: string;
+  readonly disabled?: boolean;
```

Comment:
```suggestion
  readonly disabled: boolean;
```
same here, as what said with another comment: if you like the idea to not have the `?` on a boolean, then I'd change this as well

Reply:
Also here i prefer to use optional but for different reason. In this case i have marked `disabled` attribue as optional in props but set to `false` in react component.  So disabled will be always a boolean with default value inside  `LinkButton`  component but can be omitted when used.